sbteclipse-core/src/main/scala/com/typesafe/sbteclipse/core/Eclipse.scala

```diff
@@ -678,6 +683,33 @@ private object Eclipse extends EclipseSDTConfig {
       fix(parts, "")
     } else filename
   }
+
+  implicit val fileEqual = new Equal[File] {
+    def equal(file1: File, file2: File): Boolean = file1 == file2
+  }
+
+  def id[A](a: A): A = a
+
+  def boolOpt(key: String): Parser[(String, Boolean)] = {
+    import sbt.complete.DefaultParsers._
+    (Space ~> key ~ ("=" ~> ("true" | "false"))) map { case (k, v) => k -> v.toBoolean }
+  }
+
+  def setting[A](key: SettingKey[A], state: State): Validation[A] =
```

Comment:
You should have some kind of warning/doc about this method requiring FULLY qualified key.